web/services/getPersonalSummaryByDateService.php

```diff
@@ -129,7 +129,27 @@
         }
         if ($minutes < 10)
             $minutes = "0" . $minutes;
-        $pendingHolidays = $hours . ":" . $minutes;
+
+        $currentJourney = 8;
+        $journeys = UsersFacade::GetUserJourneyHistoriesByIntervals($initDate, $date, $userVO->getId());
+        if(count($journeys)==1) {
+                $currentJourney = $journeys[0]->getJourney();
+        }
+
+        if ($hours > $currentJourney*7) {
+            $days = intval($hours/$currentJourney);
+            $hours = round($hours - ($days*$currentJourney),2);
+            if ($hours == $currentJourney) {
+                $hours = 0;
+                $days = $days + 1;
+            }
+            if ($hours < 10) {
+                $hours = "0" . $hours;
+            }
+        $pendingHolidays = intval($days) . " days " . intval($hours) . ":" . intval($minutes);
```

Comment:
You haven't re-calculated `$minutes` and this will give you wrong data if journey data is float.

Reply:
A practical example, with a journey of 7.2h and 21.6h of holidays, that's exactly three days of holidays, but your code says "2 days 6:36 h".

Reply:
Also, the indentation of this line is incorrect.

Reply:
will take a look asap!